public/assets/js/profile.js

```diff
@@ -0,0 +1,133 @@
+      }</span><div class="kwt-file__delete"></div></div></div>`
+    );
+    var element = element.parents(".kwt-file");
+
+    // Add class on focus and drage enter event.
+    element.on("dragenter focus click", ".kwt-file__input", function (e) {
+      $(this).parents(".kwt-file__drop-area").addClass("is-active");
+    });
+
+    // Remove class on blur and drage leave event.
+    element.on("dragleave blur drop", ".kwt-file__input", function (e) {
+      $(this).parents(".kwt-file__drop-area").removeClass("is-active");
+    });
+
+    // Show filename when change file.
+    element.on("change", ".kwt-file__input", function (e) {
+      let filesCount = $(this)[0].files.length;
+      let textContainer = $(this).next(".kwt-file__msg");
+      if (1 === filesCount) {
+        let fileName = $(this).val().split("\\").pop();
+        textContainer
+          .text(fileName)
+          .next(".kwt-file__delete")
+          .css("display", "block");
+      } else if (filesCount > 1) {
+        textContainer
+          .text(filesCount + " files selected")
+          .next(".kwt-file__delete")
+          .css("display", "inline-block");
+      } else {
+        textContainer.text(
+          `${
+            "" === this[0].placeholder
+              ? "or drop files here"
+              : `${this[0].placeholder}`
+          }`
+        );
+        $(this)
+          .parents(".kwt-file")
+          .find(".kwt-file__delete")
+          .css("display", "none");
+      }
+    });
+
+    // Delete selected file.
+    element.on("click", ".kwt-file__delete", function (e) {
+      let deleteElement = $(this);
+      deleteElement.parents(".kwt-file").find(`.kwt-file__input`).val(null);
+      deleteElement
+        .css("display", "none")
+        .prev(`.kwt-file__msg`)
+        .text(
+          `${
+            "" ===
+            $(this).parents(".kwt-file").find(".kwt-file__input")[0].placeholder
+              ? "or drop files here"
+              : `${
+                  $(this).parents(".kwt-file").find(".kwt-file__input")[0].placeholder
+                }`
+          }`
+        );
+    });
+  };
+
+  $.fn.kwtFileUpload = function (e) {
+    var _this = $(this);
+    $.each(_this, function (index, element) {
+      customDragandDrop(element);
+    });
+    return this;
+  };
+})(jQuery);
+
+// Plugin initialize
+jQuery(document).ready(function ($) {
+  $(".demo1").kwtFileUpload();
+});
+
+
+function enable_Input() {
+  let input = document.getElementById('bio');
+  if(input.disabled == true){
+    input.disabled = false;
+  }
+  else{
+    input.disabled = true;
+  }
+}
+
+function disable_Input(){
+  let input = document.getElementById('bio');
+  if(input.disabled == false){
+    input.disabled = true;
+  }
+
+}
+
+function enable(){
+  let input = document.getElementById('bio');
+  if(input.disabled == true){
+    input.disabled = false;
+  }
+}
+var type = document.getElementById('mode');
+function darkmode() {
```

Comment:
1/ naming convetion is incorrect
2/ Why is this function declared multiple times in different files? Couldn't you have created a single file?

Reply:
Implemented in a single file